--- glfx.js	2013-07-12 15:25:23.404796323 -0700
+++ glfx.patched.js	2013-07-12 15:26:46.448497606 -0700
@@ -182,7 +182,8 @@
     };
 
     // Go for floating point buffer textures if we can, it'll make the bokeh filter look a lot better
-    targetTextureType = gl.getExtension('OES_texture_float') ? gl.FLOAT : gl.UNSIGNED_BYTE;
+    // targetTextureType = gl.getExtension('OES_texture_float') ? gl.FLOAT : gl.UNSIGNED_BYTE;
+    targetTextureType = gl.UNSIGNED_BYTE;
     gl.getExtension("OES_texture_float_linear");
 
     // Core methods
