--- glfx.js	2013-07-12 14:43:28.561842451 -0700
+++ glfx.patched.js	2013-07-12 14:45:47.029344370 -0700
@@ -1286,20 +1286,6 @@
 function initialize(width, height) {
     var type = gl.UNSIGNED_BYTE;
 
-    // Go for floating point buffer textures if we can, it'll make the bokeh
-    // filter look a lot better. Note that on Windows, ANGLE does not let you
-    // render to a floating-point texture when linear filtering is enabled.
-    // See http://crbug.com/172278 for more information.
-    if (gl.getExtension('OES_texture_float')) {
-        var testTexture = new Texture(100, 100, gl.RGBA, gl.FLOAT);
-        try {
-            // Only use gl.FLOAT if we can render to it
-            testTexture.drawTo(function() { type = gl.FLOAT; });
-        } catch (e) {
-        }
-        testTexture.destroy();
-    }
-
     if (this._.texture) this._.texture.destroy();
     if (this._.spareTexture) this._.spareTexture.destroy();
     this.width = width;
